### DISCUSSION TITLE
SUGGESTION: Individualized MMR

### DISCUSSION THREAD

**Discussion by abester605** [ValveSoftware/Dota2-Gameplay#21934] [Ideas]
Hello - I want to start by saying that I appreciate all the work you put into making the game fun, challenging, and ever changing.  I have noticed over the years that much effort has been put into the matchmaking and ranking systems; yet, there still seems to be a discomfort and disconnect between what is operating and what would make the community happy.  The reporting system and Overwatch were huge in helping to limit teams that don't get along; but, even though this is a team game, many people still go into it individually or with one or two friends and it becomes very frustrating when you are paired with others who have their own troubles or view of the game causing major coordination issues on games that otherwise should have been fun if not victorious.  Too many of us have been in too many games where we are dominating our role but still lose because of attempts to coordinate with 4 strangers going poorly.  In light of this, I would like to suggest a new type of system (which you can tweak and modify to be more real with how you believe the game is best played) that rewards playing in your role successfully and punishes you less for a troll who just wants to bring down everyone's MMR because something happened at min 3.

5v5 games: keep the straight 25 MMR up down method as anyone in a party like this should have their own coordination and strategy down.

All other games, reward MMR based on metrics.  For ranked roles, you might limit to playing within the roles or you may make the benefits available to not have to assume roles in a game that is not predetermined.  Breaking down by roles, here is the thought:

General:  
   plus 5 MMR for win 
   minus 10 MMR loss
   minus 1 MMR for every 1 death

Core
   plus  1 MMR for every 1 kill
   plus  1 MMR for every 3 assists
   plus  1 MMR for every 7,500 damage done (net)
   plus  1 MMR for every 15,000 damage taken (gross) [of course, dying to take damage won't help - this is to reward the offlane]

Support
   plus  1 MMR for every 30 seconds ward provides vision of an enemy
   plus  1 MMR for every observer ward countered
   plus  1 MMR for every 8,000 damage healed
   plus  1 MMR for every 2 assists (if you can break it up, otherwise tweak the heal or ward vision)
   plus  1 MMR for every 60 seconds of enemy CC or ally buff (maybe count force staff as 3 seconds - don't count buff on self)

The goals of this would be to limit trolling as it will be more detrimental to yourself than to the people you are mad at.  Reward those who dominate while diminishing the reward for those who are carried and don't participate.  Provide some relief and growth opportunity for when you are playing your role well; but, can't seem to find that team to win...  you can still work your way up to a more true rank for your skill level.  It may also be a good idea to undo some of the other matchmaking tricks (like pairing the lost streak people with the win streak people); but, hopefully the system will work well enough that you can keep that and people will still be able to earn their rank despite PUG challenges.

Thank you.

Abester

**Comment by ghost** [comment]
Some heroes such as Techies have to face death. Could this be a balance counting for heroes that require sacrifice or tanking during the team wars? IMO, KDA shouldn't be taken as a factor to adjust the MMR because the KDA in every game cannot be taken as any factor to be analysed as a player's performance, which means your KDA is low, medium, or high. It's still the same as the KDA of other heroes who may carry different values during the post-game.

**Comment by commentator24** [comment]
this sounds like a very trolling idea.. why bother winning or losing then? just make the game very long and spam armlet + healing heroes or forcestaff each other.
![image](https://github.com/user-attachments/assets/6e1ebcf3-6fd0-4259-b634-a6f64bc6d5ae)
here losing team can get 42 mmr by core standards..?
